### PR TITLE
GC dataformat-xml

### PR DESCRIPTION
**Pull request by baldersheim**
@bjorncs or @gjoranv PR

### CODE REVIEW

File: configserver/pom.xml [hunk 1]
```diff
@@ -291,11 +291,6 @@
         <groupId>com.yahoo.vespa</groupId>
         <artifactId>bundle-plugin</artifactId>
         <extensions>true</extensions>
-        <configuration>
-          <!-- TODO: Vespa > 8: remove importPackage when the jackson-jaxrs-json-provider bundle is no longer installed in jdisc -->
-          <!--   added to ensure using the same class as orchestrator, core-dump-reporter, provision-controller and controller-clients -->
-          <importPackage>com.fasterxml.jackson.jaxrs.json;version="[2.12.6,3)"</importPackage>
-        </configuration>
```
Comment (bjorncs) [inline configserver/pom.xml:298]: @gjoranv Is this related to dataformat-xml?

Comment (gjoranv) [reply configserver/pom.xml:298]: No, this cannot be removed now.

**Review by gjoranv — CHANGES_REQUESTED** [review]
Great, except for the change in configserver.

File: configserver/pom.xml (hunk 1 above)
Comment (baldersheim) [reply configserver/pom.xml]: Reverted this change back.

**Review by gjoranv — APPROVED** [review]
